GooglePlacesAutocomplete.js

```diff
@@ -216,10 +218,19 @@ const GooglePlacesAutocomplete = React.createClass({
   },
 
   getCurrentLocation() {
-    let options = (Platform.OS === 'android') ? {enableHighAccuracy: true, timeout: 20000} : {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000};
+    let options = null;
+    if (this.props.enableHighAccuracyLocation)
+      options = (Platform.OS === 'android') ? {enableHighAccuracy: true, timeout: 20000} : {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000};
     navigator.geolocation.getCurrentPosition(
       (position) => {
-        this._requestNearby(position.coords.latitude, position.coords.longitude);
+        if (this.props.nearbyPlacesAPI === 'None') {
```

Comment:
Hi,

Can't we add this option 'None' like you did with the other two ?

The thing is, I want to be able to catch current location coords through the onPress function. However I don't want to call any API, thus the 'None' option, 'cause I want the use of current location but not the use of related APIs.

In this state, the option 'None' I added doesn't conflict with the rest and it'll work as intended and like it used to.

Is that ok with you?

Thanks in advance for your reply.